### ISSUE TITLE
Failing to fetch url

### ISSUE THREAD

**Issue by codercatdev** [elasticio/odata2openapi#11]
This works in the code
```
    odata2openapi('http://services.odata.org/V3/Northwind/Northwind.svc/$metadata')
      .then(swagger => console.log(JSON.stringify(swagger, null, 2)))
      .catch(error => console.error(error))
```

This will not
```
odata2openapi('http://becci01.haworthinc.com:8000/sap/opu/odata/sap/ZGW0001V1_TECHNICAL_SRV/$metadata')
      .then(swagger => console.log(JSON.stringify(swagger, null, 2)))
      .catch(error => console.error(error))
```
and ends in this error
```
Failed to execute 'fetch' on 'Window': Failed to parse URL from //[becci01.haworthinc.com:8000]/sap/opu/odata/sap/SD_MYCONTACTS/$metadata
```

**Comment by codercatdev** [comment]
In test this is fixed by pull request #1

**Comment by therealmitchconnors** [comment]
I think this is a duplicate of issue #12, which is fixed by pull request #14 .  Thoughts?

**Comment by akorchev** [comment]
Sorry for the late response. I've expressed my thoughts on the matter here: https://github.com/akorchev/odata2openapi/issues/12#issuecomment-344315701
